observations/obs_converters/GSI2DART/README

```diff
@@ -11,22 +11,30 @@ are placed under the directory enkf
    kinds.F90
    mpi_readobs.f90
    read_diag.f90
+
```

Comment:
Regarding the paths in lines 4-6:

1. Where is the `GSI` directory? Is it the `${DARTROOT}/observations/obs_converters/GSI2DART directory`?
2. If so should we change the path from GSI to GSI2DART?